From 931ff9b00528c4a8a7e81c3b39353047d32f181e Mon Sep 17 00:00:00 2001
From: Nathan Rossi <nathan@nathanrossi.com>
Date: Mon, 26 Aug 2019 21:27:04 +1000
Subject: [PATCH] Fix message pattern in gcc testsuite

Change the dg-message pattern to handle quotes in the message.

gcc/testsuite/ChangeLog:

2019-08-26  Nathan Rossi  <nathan@nathanrossi.com>

        gcc.target/arm/polytypes.c: Fix pattern.

Upstream-Status: Pending
---
 gcc/testsuite/gcc.target/arm/polytypes.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/testsuite/gcc.target/arm/polytypes.c b/gcc/testsuite/gcc.target/arm/polytypes.c
index 110d62a879..3d49b508a8 100644
--- a/gcc/testsuite/gcc.target/arm/polytypes.c
+++ b/gcc/testsuite/gcc.target/arm/polytypes.c
@@ -28,7 +28,7 @@ void foo ()
   poly8x16_t v128_8;
   poly16x8_t v128_16;
 
-  s64_8 (v64_8); /* { dg-message "use -flax-vector-conversions" } */
+  s64_8 (v64_8); /* { dg-message "use '?-flax-vector-conversions" } */
   /* { dg-error "incompatible type for argument 1 of 's64_8'" "" { target *-*-* } .-1 } */
   u64_8 (v64_8); /* { dg-error "incompatible type for argument 1 of 'u64_8'" } */
   p64_8 (v64_8);
